src/Clapi/TimeDelta.hs

```diff
@@ -0,0 +1,22 @@
+module Clapi.TimeDelta (TimeDelta, getDelta, tdZero, tdClVal) where
+import Data.Word (Word32)
+import System.Clock (Clock(Monotonic), TimeSpec, getTime, toNanoSecs)
+
+import Clapi.Types (Time(..), TimeStamped(..), ClapiValue(ClFloat))
+
+timeToFloat :: Time -> Float
+timeToFloat (Time s f) = fromRational $ toRational s + (toRational f / toRational (maxBound :: Word32))
+
+getMonotonicTimeFloat :: IO Float
+getMonotonicTimeFloat = (\ts -> fromRational $ (toRational $ toNanoSecs ts) / 10.0^9) <$> getTime Monotonic
```

Comment:
Likewise, this might be clearer as `getTime Monotonic >>= return . fromRational . (/ 10.0^9) . toRational . toNanoSecs` (although please check my logic). Again, this means I don't have to jump around when reading the definition, especially in working on when that division applies...